### DISCUSSION TITLE
liveURL implementation

### DISCUSSION THREAD

**Discussion by vlrevolution** [browserless/chrome#3987] [Q&A] [answered]
I am very interested in the liveURL implementation of your enterprise offering but I am a solo dev and don't really qualify as an enterprise for now. I'd like to ask if there is any functionality built into the library that makes the liveURL feature possible? I'd love to be able to have a hybrid session with puppeteer, any info you could give me to get this working myself for my passion project?

**Comment by joelgriffith** [comment, marked as answer]
Sadly unavailable for now due to some proprietary code. We will talk a bit about this at the conference coming up soon!  https://www.browserconference.com/talks/user-in-the-loop/

**Comment by DenisBessa** [comment]
As a solo developer myself, I agree with the OP. It would be an awesome feature.
